CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Azure Kubernetes Service Changelog
 
+
+## Release 2024-04-11
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+- [Dual-stack networking (IPv4/IPv6) with CNI Overlay](https://learn.microsoft.com/en-us/azure/aks/azure-cni-overlay?tabs=kubectl#dual-stack-networking) is now generally available (GA) for Linux node pools and does not need the Azure Feature Flag registered on the subscription.
```

Comment:
this needs to go under Release Notes -> Features section as it's a GA of a new capability. And we need to remove 'en-us' for localization friendliness